### ISSUE TITLE
crashing after update to 2.5.7, can't do anything to get my calendar back

### ISSUE THREAD

**Issue by tpo** [SimpleMobileTools/Simple-Calendar#195]
So, after upgrading to 2.5.7 via F-Droid Simple-Calendar is crashing immediately upon start.

That is I am currently without my calendar, which is quite bad.

* So I tried to downgrade to 2.5.6 or 2.5.3 (that's the version I upgraded from). Result: "Fehler beim Installieren von Calendar. Installation wegen unbekanntem Fehler fehlgeschlagen" - Translation: "Error while installing Calendar. Installation aborted due to unknown error"

* I tried to upgrade via Google Play to the newest version that's available there (2.6.1). I can't because Google Play recognizes that "Simple-Calendar" ist installed, however it apparently doesn't recognize that I should be able to upgrade to the version available there and thus there is no "Install" button but just an "Open" button.

* then I tried to download the latest APK from Github and install that. However when I do that (i.e. I "open" the APK file from the "Downloads") then the installer (I guess the Google installer) tells me that "The application is allready installed with a different signature". All I can then do is to press "OK" and that's it.

* I tried to find where "Simple-Calendar" is saving its calendar with Ghost Commander (File Manager, available via F-Droid) but there doesn't seem to be any place where I could find that file (it's not a rooted device so I can not browse the whole Filesystem)

* I tried to export the calenda with either "Calendar ICS adapter" or with "Calendar Import-Export" (both available in F-Droid). I can't access the first one because "Simple-Calendar" is crashing, so I can't even get to the import-export step. "Calendar Import-Export" doesn't seem to list "Simple-Calendar"'s calendar, so I can not export it.

The last thing I could do is to deinstall "Simple-Calendar" and then to install it again, but then I guess (I don't know - how can I find out?) I'll loose all of my calendar data, which is exactly the point why I don't want to do that.

I think you might want to reconsider #188 because it's not "that was just a hotfix fixing some issues with people unable to start the app after update", but "some pleople actually completely lost access to their calendar at this point and can't get the access back".

So to recap, at this point I'm out of ideas how I can get back my calendar data or a working "Simple-Calendar" app. Any ideas?

Also, if that's of any motivation, I am ready to actually pay for support to get my problem fixed. Please let me know.

Thanks,
*t

**Comment by tibbi** [comment]
Hey,
you cannot open the calendar either if you create a desktop widget and try opening it through it?
F-droid is apparently failing to build the latest version for some weird error that their build script is having. They are signing the builds with their own keys, so I just cannot make a new build which would upgrade properly. The events are in a database, you cannot find them with a file manager.
Google sync has been added meanwhile, Ill take a look at home if I can squeeze out some new calendar build or find out whats wrong, as they are building it themselves with some automated scripts.

**Comment by tpo** [comment]
> you cannot open the calendar either if you create a desktop widget and try opening it through it?

Woah, that works. Phew, I'm relieved. Thanks for the tip!!!

I wrote:

> I tried to export the calendar with [...] with "Calendar Import-Export" (available in F-Droid). [... It] doesn't seem to list "Simple-Calendar"'s calendar, so I can not export it.

Do you have an idea why that is? I'd think "Simple-Calendar" also uses the calendar API (I'm guessing) to save calendar entries and so "Calendar Import-Export" should be able to see "Simple-Calendar"'s calendar?

Again, *many thanks*,
*t

**Comment by tibbi** [comment]
Ok, glad it works :) My calendar doesnt have Caldav sync yet, which means that the events are stored only in the Simple Calendar alone, without the system knowing that its actually a calendar app with reusable events. Adding Caldav sync for sharing events between calendars is on the roadmap.
